### PR TITLE
Translation + license update

### PR DESCRIPTION
**Pull request by fdechelle**
- license changed to CC BY-SA
- glossary translated

### CODE REVIEW

**Comment by cconard96** [comment]
Build is visible at: https://glpi-user-documentation--44.org.readthedocs.build/fr/44

File: source/glossary.rst [hunk 1]
```diff
@@ -0,0 +1,418 @@
...
+   Registered user
+      Person having an accreditation allowing to connect to GLPI.
+
+   Contact
+      Referent person linked to a supplier.
+
+   Contract
+      Financial document linked to assets and to suppliers
+
+   Controler
```
Comment (cconard96) [inline source/glossary.rst:65]:
```suggestion
   Controller
```

File: source/glossary.rst [hunk 2]
```diff
@@ -0,0 +1,418 @@
...
+
+      .. ??? Domaine de messagerie
+
+   Domain
+      Grouping of assets connected to a network.
+
+      .. ??. Regroupement de matériels connectés au réseau.
+
+   Title
+      Customizable element of a a drop-down list
```
Comment (cconard96) [inline source/glossary.rst:105]:
```suggestion
      Customizable element of a drop-down list
```
I don't think this is called Title anywhere in GLPI under the default language.

File: source/glossary.rst [hunk 3]
```diff
@@ -0,0 +1,418 @@
...
+   Supervisor
+      Person in charge of a group of users.
+
+   Provider
+      Legal structure toward which a contractual relation can be established (purchase, contract...). 
+
+   VLAN
+      :abbr:`VLAN (Virtual Local Area Network)` : Virtual Local Network.
+
+   VLAN Taggé
```
Comment (cconard96) [inline source/glossary.rst:352]:
```suggestion
   Tagged VLAN
```

File: source/glossary.rst [hunk 4]
```diff
@@ -0,0 +1,418 @@
...
+   Template
+      Reusable model of object containing predefined fieds and allowing easier user entry.
+
+   Tier type
+      Category of providers.
+
+   Ticket
+      GLPI object representing an incident or a service request.
+
+   Transfert
```
Comment (cconard96) [inline source/glossary.rst:373]:
```suggestion
   Transfer
```

File: source/glossary.rst (hunk 2 above)
Comment (fdechelle) [reply source/glossary.rst:105]: In French it is "intitulé", which can be translated as "Entitled", "Named", "Designation". Don't know how it is called in GLPI UI.

Comment (cconard96) [reply source/glossary.rst:105]: It is just called "Dropdowns" in the menu for en_GB and en_US.

File: source/glossary.rst [hunk 5]
```diff
@@ -0,0 +1,418 @@
...
+   Accepted
+      State of a demand which has been validated by an approver.
+
+   Actions
+      Grouping in a list of available handling of GLPI objects. 
+
+   Adaptive rule
+      Rule in which at least one action is the result of a regular expression defined in a criteria.
+
+      .. ??? Règle dont au moins une action est le résultat d'une expression rationnelle définie dans un critère.
```
Comment (trasher) [inline source/glossary.rst:16]:
```suggestion
```

File: source/glossary.rst [hunk 6]
```diff
@@ -0,0 +1,418 @@
...
+   Case
+      Physical element that contains a computer's components (motherboard, processor, disk...).
+
+   SSO
+      Unique authentication system (:abbr:`SSO (Single Sign On)`).
+
+   Accreditation
+      Authorization to perform a set of actions on several entities, given by rights attribution process.
+
+      .. ??? Autorisation résultante de l'attribution de droits sur un périmètre d'action (une ou plusieurs entités).
```
Comment (trasher) [inline source/glossary.rst:48]:
```suggestion
```

File: source/glossary.rst [hunk 7]
```diff
@@ -0,0 +1,418 @@
...
+   Contact
+      Referent person linked to a supplier.
+
+   Contract
+      Financial document linked to assets and to suppliers
+
+   Controller
+      IT component
+
+      .. ??? Composant informatique.
```
Comment (trasher) [inline source/glossary.rst:68]:
```suggestion
      Device that interfaces with other peripheral device.
```

File: source/glossary.rst [hunk 8]
```diff
@@ -0,0 +1,418 @@
...
+      Directory service using the :abbr:`LDAP protocol (Lightweight Directory Access Protocol)`. Allows in particular to store user accounts.
+
+   Document
+      Element that defines a link and/or store a file and that can be associated to other objects of GLPI.
+
+   Mail domain
+      Part of a mail adress that is after character `@`.
+
+      .. ??? Domaine de messagerie
+
```
Comment (trasher) [inline source/glossary.rst:98]:
```suggestion
```

File: source/glossary.rst [hunk 9]
```diff
@@ -0,0 +1,418 @@
...
+   Domain
+      Grouping of assets connected to a network.
+
+      .. ??. Regroupement de matériels connectés au réseau.
```
Comment (trasher) [inline source/glossary.rst:102]:
```suggestion
      Group of assets connected to a network.
```

File: source/glossary.rst [hunk 10]
```diff
@@ -0,0 +1,418 @@
...
+
+   Title
+      Customizable element of a a drop-down list
+
+   Backup
+      Operation that consists in duplicating and securing data.
+
+      .. ??? in glossary? seriously? Opération qui consiste à dupliquer et à mettre en sécurité les données.
+
```
Comment (trasher) [inline source/glossary.rst:111]:
```suggestion
```

File: source/glossary.rst [hunk 11]
```diff
@@ -0,0 +1,418 @@
...
+   Dynamic display
+      Mode that offers a better user experience in using GLPI thanks to Ajax.
+
```
Comment (trasher) [inline source/glossary.rst:114]:
```suggestion
```

File: source/glossary.rst [hunk 12]
```diff
@@ -0,0 +1,418 @@
...
+   Writer
+      Person writing.
+
+      .. ??? in glossary? seriously?
```
Comment (trasher) [inline source/glossary.rst:118]:
```suggestion
```

File: source/glossary.rst [hunk 13]
```diff
@@ -0,0 +1,418 @@
...
+
+   External link
+      Link displayed on an asset form in GLPI and built dynamically using the asset data.
+      
+   Frequently Asked Questions (FAQ)
+      Specifically in GLPI, selection of elements of the knowledge base that can be provided independently, for example articles that are made accessible to users having only access to the simplified interface.
+
+   Field
+      Element of the graphical interface that allows an interaction with the user (selection, entry...).
+
```
Comment (trasher) [inline source/glossary.rst:137]: Not sure this is relevant in glossary
```suggestion
```

File: source/glossary.rst [hunk 14]
```diff
@@ -0,0 +1,418 @@
...
+   Pending (Ticket)
+      Status of a ticket which processing is temporarily suspended (does not impact processing time).
+
+   Webcal
+      Exchange format for calendar data.
+
+   SLA
+      SLA (for Service-Level Agreement) is a contract between a service provider and a client that defines the requested quality of service and the responsibilities in case of failure.
+
+      .. ??? charabia Le service-level agreement (SLA) ou « accord de niveau de service » est un document qui définit la qualité de service, prestation prescrite entre un fournisseur de service et un client. Autrement dit, il s'agit de clauses basées sur un contrat définissant les objectifs précis attendus et le niveau de service que souhaite obtenir un client de la part du prestataire et fixe les responsabilités.
```
Comment (trasher) [inline source/glossary.rst:418]:
```suggestion
```

File: source/glossary.rst (hunk 2 above)
Comment (trasher) [reply source/glossary.rst:105]: Maybe something like:
```
   Dropdown
      Customizable dropdowns list used in GLPI and plugins.
```

File: source/glossary.rst (hunk 13 above)
Comment (fdechelle) [reply source/glossary.rst:137]: I agree, it is not relevant.

File: source/glossary.rst (hunk 2 above)
Comment (fdechelle) [reply source/glossary.rst]: Agree and changed.

**Review by trasher — APPROVED** [review]

**Review by orthagh — APPROVED** [review]

**Pull request merged**